src/mixins/amalgamation-mixin.ts

```diff
@@ -200,9 +205,35 @@ export default class AmalgamationMixin extends Vue {
     )
   }
 
+  /**
+   * This business is future effective if the first filing in the ledger is future effective and is still
+   * not complete or corrected (ie, it's paid or pending).
+   * @param business The business to check if is Future Effective or not.
+   */
+  isFutureEffective (business: any): boolean {
+    return (
+      business.firstFiling?.isFutureEffective === true &&
+      business.firstFiling?.status !== FilingStatus.COMPLETED &&
+      business.firstFiling?.status !== FilingStatus.CORRECTED
```

Comment:
Note the small change from before -- previous we check status for PAID or PENDING. This new check is consistent with what we do in [Filings UI](https://github.com/bcgov/business-filings-ui/blob/4f6698f3f4fb6e95fef6c4695ac3abb31b2afb21/src/components/Dashboard/FilingHistoryList/HeaderActions.vue#L123) (ie, reuse business logic we already thought about).